.github/workflows/ci.yaml

```diff
@@ -53,14 +53,16 @@ jobs:
         charm:
           - path: .
             charmcraft: 2.x/stable
+            cache: true
           - path: tests/integration/app-charm
             charmcraft: latest/stable
+            cache: false
```

Comment:
Test app builds on charmcraft 3, due to strict binary dependencies, but that breaks charmcraftcache-hub